packages/odata-service-inquirer/src/prompts/edmx/questions.ts

```diff
@@ -312,14 +317,20 @@ function getTableLayoutQuestions(
                 breadcrumb: true,
                 mandatory: true
             },
-            default: '',
+            default: (prevAnswers: EntitySelectionAnswers & TableConfigAnswers) => {
+                // Auto-populate qualifier from RecursiveHierarchy annotation if available
+                const entitySetName = prevAnswers?.mainEntity?.entitySetName;
```

Comment:
To improve understandability and readability  of the code (multiple ternaries make for hard to read code), and use less variables: 
```suggestion
                if (prevAnswers?.mainEntity?.entitySetName) {
                    const entitySet = findEntitySetByName(metadata, entitySetName);
                    return entitySet ? getRecursiveHierarchyQualifierForEntitySet(entitySet) : ''
                }
                return '';
```